enterprise/replay/setup.go

```diff
@@ -23,7 +23,7 @@ func loadConfig() {
 }
 
 func initFileManager() (filemanager.FileManager, string, error) {
-	bucket := strings.TrimSpace(config.GetString("JOBS_BACKUP_BUCKET", ""))
+	bucket := strings.TrimSpace(config.GetString("JOBS_REPLAY_BACKUP_BUCKET", ""))
```

Comment:
@cisse21 Is it OK to just change this configuration setting? Doesn't it have to be backward compatible or something?

Reply:
Replay is done on demand. These envs are not served anywhere. So, it is okay for now.